斐讯K2P编译失败

## 1.关于你要提交的问题

Q：是否搜索了issue (使用 "x" 选择)
* [x] 没有类似的issue

## 2. 详细叙述
斐讯K2P / MT7621 编译失败

### (1) 具体问题

A：斐讯K2P编译失败，似乎是与新增加的Redmi AC2100支持有关，Revert commit [
01aeb5d
](https://github.com/coolsnowwolf/lede/commit/01aeb5df0ed4c2119c9cbdb072872089d4739a58)之后恢复正常。

### (2) 路由器型号和固件版本

A：Phicomm K2P / R20.4.8

### (3) 详细日志

A：部分log:
```
$ ionice -c 3 nice -n 20 make -j8 V=s CONFIG_DEBUG_SECTION_MISMATCH=y 2>&1 | tee build.log
.....
/tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/linux-4.14.176/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg  -o /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_xzwifi_creativebox-v1.dtb /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_xzwifi_creativebox-v1.dtb.tmp
rm -f /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_xiaomi_mir3p.dtb.tmp
/tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_xiaomi_redmi-router-ac2100.dtb: ERROR (phandle_references): Reference to non-existent node or label "gpio"

mipsel-openwrt-linux-musl-cpp -nostdinc -x assembler-with-cpp -I/tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/linux-4.14.176/arch/mips/boot/dts -I/tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/linux-4.14.176/arch/mips/boot/dts/include -I/tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/linux-4.14.176/include/ -undef -D__DTS__  -o /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_zbtlink_zbt-we1326.dtb.tmp ../dts/mt7621_zbtlink_zbt-we1326.dts
/tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_xiaomi_redmi-router-ac2100.dtb: ERROR (phandle_references): Reference to non-existent node or label "gpio"

/tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_xiaomi_redmi-router-ac2100.dtb: ERROR (phandle_references): Reference to non-existent node or label "gpio"

/tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_xiaomi_redmi-router-ac2100.dtb: ERROR (phandle_references): Reference to non-existent node or label "gpio"

/tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_xiaomi_redmi-router-ac2100.dtb: ERROR (phandle_references): Reference to non-existent node or label "gpio"

ERROR: Input tree has errors, aborting (use -f to force output)
make[5]: *** [Makefile:202: /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_xiaomi_redmi-router-ac2100.dtb] Error 2
make[5]: *** Waiting for unfinished jobs....
rm -f /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_xiaomi_mir3g-v2.dtb.tmp
/tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/linux-4.14.176/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg  -o /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_youhua_wr1200js.dtb /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_youhua_wr1200js.dtb.tmp
/tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/linux-4.14.176/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg  -o /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_youku_yk-l2.dtb /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_youku_yk-l2.dtb.tmp
rm -f /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_xiaoyu_xy-c5.dtb.tmp
rm -f /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_xzwifi_creativebox-v1.dtb.tmp
rm -f /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_youhua_wr1200js.dtb.tmp
/tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/linux-4.14.176/scripts/dtc/dtc -O dtb -i../dts/ -Wno-unit_address_vs_reg -Wno-simple_bus_reg -Wno-unit_address_format -Wno-pci_bridge -Wno-pci_device_bus_num -Wno-pci_device_reg  -o /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_zbtlink_zbt-we1326.dtb /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_zbtlink_zbt-we1326.dtb.tmp
rm -f /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_youku_yk-l2.dtb.tmp
rm -f /tmp/lede/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/image-mt7621_zbtlink_zbt-we1326.dtb.tmp
make[5]: Leaving directory '/tmp/lede/target/linux/ramips/image'
make[4]: *** [Makefile:26: install] Error 2
make[4]: Leaving directory '/tmp/lede/target/linux/ramips'
make[3]: *** [Makefile:13: install] Error 2
make[3]: Leaving directory '/tmp/lede/target/linux'
time: target/linux/install#54.58#16.21#48.83
make[2]: *** [target/Makefile:25: target/linux/install] Error 2
make[2]: Leaving directory '/tmp/lede'
make[1]: *** [target/Makefile:19: /tmp/lede/staging_dir/target-mipsel_24kc_musl/stamp/.target_install] Error 2
make[1]: Leaving directory '/tmp/lede'
make: *** [/tmp/lede/include/toplevel.mk:227：world] 错误 2
```
config: https://github.com/KevinMX/openwrt-phicomm-k2p-build/blob/master/.config
完整log:
[build.log](https://github.com/coolsnowwolf/lede/files/4563918/build.log)

--- a/target/linux/ramips/dts/mt7621_xiaomi_redmi-router-ac2100.dts
+++ b/target/linux/ramips/dts/mt7621_xiaomi_redmi-router-ac2100.dts
@@ -26,22 +26,22 @@
 
 		led_status_amber: status_amber {
 			label = "rm2100:amber:status";
-			gpios = <&gpio 6 GPIO_ACTIVE_LOW>;
+			gpios = <&gpio0 6 GPIO_ACTIVE_LOW>;
 		};
 
 		led_status_white: status_white {
 			label = "rm2100:white:status";
-			gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+			gpios = <&gpio0 8 GPIO_ACTIVE_LOW>;
 		};
 
 		wan_amber {
 			label = "rm2100:amber:wan";
-			gpios = <&gpio 10 GPIO_ACTIVE_LOW>;
+			gpios = <&gpio0 10 GPIO_ACTIVE_LOW>;
 		};
 
 		wan_white {
 			label = "rm2100:white:wan";
-			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
 		};
 	};
 
@@ -50,7 +50,7 @@
 
 		reset {
 			label = "reset";
-			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+			gpios = <&gpio0 18 GPIO_ACTIVE_LOW>;
 			linux,code = <KEY_RESTART>;
 		};
 	};

